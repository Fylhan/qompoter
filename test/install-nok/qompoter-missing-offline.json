{
	"name": "test/install-nok-missing",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.0",
		"missing/package": "dev-master",
		"fylhan/posibrain": "v0.2"
	}
}
