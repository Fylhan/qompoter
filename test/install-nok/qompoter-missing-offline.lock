{
  "name": "test/install-nok-missing",
  "date": "2016-12-02T18:24:51+01:00",
  "require": {
    "cp/fylhan": {"version": "v1.0", "type": "qompoter-fs", "url": "", "md5sum": "ea9f548ff879e721c12085d7cddce551"},
  }
}
