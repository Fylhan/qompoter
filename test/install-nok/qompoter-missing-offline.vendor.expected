vendor
vendor/fylhan
vendor/fylhan/qompoter.json
vendor/fylhan/qompoter.pri
vendor/qompote.pri
